src/hardware/vga_draw.cpp

```diff
@@ -1697,53 +1759,14 @@ void VGA_SetupDrawing(uint32_t /*val*/)
 		}
 	}
 
-	/*
-      6  Horizontal Sync Polarity. Negative if set
-      7  Vertical Sync Polarity. Negative if set
-         Bit 6-7 indicates the number of lines on the display:
-            1:  400, 2: 350, 3: 480
-	*/
-	//Try to determine the pixel size, aspect correct is based around square pixels
-
-	//Base pixel width around 100 clocks horizontal
-	//For 9 pixel text modes this should be changed, but we don't support that anyway :)
-	//Seems regular vga only listens to the 9 char pixel mode with character mode enabled
-	const auto pwidth = 100.0 / static_cast<double>(htotal);
-	// Base pixel height around vertical totals of modes that have 100
-	// clocks horizontal Different sync values gives different scaling of
-	// the whole vertical range VGA monitor just seems to thighten or widen
-	// the whole vertical range
-	double pheight;
-	double target_total = 449.0;
-	Bitu sync = vga.misc_output >> 6;
-	const auto f_vtotal = static_cast<double>(vtotal);
-	switch ( sync ) {
-	case 0:		// This is not defined in vga specs,
-				// Kiet, seems to be slightly less than 350 on my monitor
-		//340 line mode, filled with 449 total
-		pheight = (480.0 / 340.0) * (target_total / f_vtotal);
-		break;
-	case 1:		//400 line mode, filled with 449 total
-		pheight = (480.0 / 400.0) * (target_total / f_vtotal);
-		break;
-	case 2:		//350 line mode, filled with 449 total
-		//This mode seems to get regular 640x400 timing and goes for a loong retrace
-		//Depends on the monitor to stretch the screen
-		pheight = (480.0 / 350.0) * (target_total / f_vtotal);
-		break;
-	case 3:		//480 line mode, filled with 525 total
-	default:
-		//Allow 527 total ModeX to have exact 1:1 aspect
-		target_total = (vga.mode == M_VGA && f_vtotal == 527) ? 527.0 : 525.0;
-		pheight = (480.0 / 480.0) * (target_total / f_vtotal);
-		break;
-	}
+	Fraction pixel_aspect_ratio = {1};
```

Comment:
wow!

Reply:
Nah, I just extracted it into a function, it's not gone, definitely still needed.